### DISCUSSION TITLE
is getting ev charging history possbile ? (IX1)

### DISCUSSION THREAD

**Discussion by Cacsjep** [bimmerconnected/bimmer_connected#656] [Q&A]
Thanks

**Comment by mdewendt** [comment]
i want the same but to my research it is currently not possible.

**Reply by Cacsjep** [reply]
Thanks for notifying

**Comment by curious2improve** [comment]
I think it is possible in principle, the ioBroker plugin code seems to support it (and so does the BMW app, so there must be some data source obviously).

Check here:
https://github.com/search?q=repo%3ATA2k%2FioBroker.bmw%20charging-s&type=code

**Comment by curious2improve** [comment]
In addition: not as easy to access as an API (has a delay of up to 24 hours)
You can request you car data from https://www.bmw.de/de-de/mybmw/vehicle-overview 

This will provide a ZIP file which also contains a json file with the cars complete charging history - quite interesting to know that they can track this, but also interesting to play around with if you want to generate some statistics, etc.
